### DISCUSSION TITLE
How to download workshop mods?

### DISCUSSION THREAD

**Discussion by eocron** [SteamRE/SteamKit#1211] [General]
Is there a way to download mods through this framework? Current SteamWorkshop handler doesn't have anything other than listing published files.

**Comment by psychonic** [comment]
DepotDownloder has examples of this, utilizing SteamKit.

https://github.com/SteamRE/DepotDownloader

Downloading SteamPipe/ISteamUGC workshop content (which covers most items in most games) is extremely similar to downloading game content.
